client-app/pages/product.vue

```diff
@@ -1,138 +1,147 @@
-        <component
-          :is="productInfoSection?.type"
-          v-if="productInfoSection && !productInfoSection.hidden"
-          :product="product"
-          :variations="variations"
-          :model="productInfoSection"
-          :fetching-variations="fetchingVariations"
-        />
+  <template v-if="product && productTemplate">
+    <VcContainer
+      ref="productComponentAnchor"
+      class="print:min-w-[1024px] print:bg-transparent print:px-0 print:[zoom:0.7]"
+    >
+      <FiltersPopupSidebar
+        :is-exist-selected-facets="hasSelectedFacets"
+        :popup-sidebar-filters="productsFilters"
+        :facets-loading="fetchingFacets"
+        :is-mobile="isMobile"
+        :is-visible="isFiltersSidebarVisible"
+        :loading="fetchingVariations"
+        :hide-controls="false"
+        @hide-popup-sidebar="hideFiltersSidebar"
+        @reset-facet-filters="resetFacetFilters"
+        @apply-filters="applyFilters"
+      />
 
-        <component
-          :is="productDescriptionSection && 'type' in productDescriptionSection ? productDescriptionSection.type : ''"
-          v-if="productDescriptionSection && !productDescriptionSection.hidden"
-          :product="product"
-          :model="productDescriptionSection"
-          :is-collapsible="false"
-        />
+      <!-- Breadcrumbs -->
+      <VcBreadcrumbs class="mb-3" :items="breadcrumbs" />
+
+      <VcTypography tag="h1">
+        {{ selectedVariationName || product.name }}
+      </VcTypography>
+
+      <div class="mt-2 flex flex-wrap gap-1 max-sm:justify-between sm:gap-6">
+        <VcCopyText
+          v-if="!product.hasVariations"
+          :text="product.code"
+          :notification="$t('pages.product.sku_copied_message')"
+        >
+          <span class="text-base text-secondary-900">
+            {{ $t("pages.product.sku_label") }}
+            <span class="font-black">#{{ product.code }}</span>
+          </span>
+        </VcCopyText>
+
+        <ProductRating v-if="productReviewsEnabled && product.rating" :rating="product.rating" />
+      </div>
 
-        <ProductConfiguration
-          v-if="product.isConfigurable && configuration?.length"
-          :product-id="productId"
-          :configuration="configuration"
-          :initial-configuration="initialConfiguration"
-        />
+      <VcLayout sidebar-position="right" sticky-sidebar class="mt-5">
+        <div class="space-y-5 xl:space-y-6">
+          <component
+            :is="productInfoSection?.type"
+            v-if="productInfoSection && !productInfoSection.hidden"
+            :product="product"
+            :variations="variations"
+            :model="productInfoSection"
+            :fetching-variations="fetchingVariations"
+          />
+
+          <component
+            :is="productDescriptionSection && 'type' in productDescriptionSection ? productDescriptionSection.type : ''"
+            v-if="productDescriptionSection && !productDescriptionSection.hidden"
```

Comment:
is it possible to make it consistent with the implementation above?
```suggestion
            :is="productDescriptionSection?.type"
            v-if="productDescriptionSection && !productDescriptionSection.hidden"
```